addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComHomeConfortTest.java

```diff
@@ -8,20 +8,40 @@
  */
 package org.openhab.binding.rfxcom.internal.messages;
 
+import static org.junit.Assert.assertArrayEquals;
 import static org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage.PacketType.HOME_CONFORT;
 
+import javax.xml.bind.DatatypeConverter;
+
 import org.junit.Test;
-import org.openhab.binding.rfxcom.internal.exceptions.RFXComMessageNotImplementedException;
+
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+import org.openhab.binding.rfxcom.internal.messages.RFXComHomeConfortMessage;
+import org.openhab.binding.rfxcom.internal.messages.RFXComHomeConfortMessage.SubType;
+import org.openhab.binding.rfxcom.internal.messages.RFXComHomeConfortMessage.Commands;
+
 
 /**
  * Test for RFXCom-binding
  *
  * @author Martin van Wingerden
+ * @author Mike Jagdis
```

Comment:
Maybe change mine to initial contribution of empty test and add after your name that you added the real test